arch/Makefile.archer2_openmpi+openmp

```diff
@@ -0,0 +1,54 @@
+# HQ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+# HQ X
+# H0 X   libAtoms+QUIP: atomistic simulation library
+# HQ X
+# HQ X   Portions of this code were written by
+# HQ X     Tamas K. Stenczel
+# HQ X
+# HQ X   Copyright 2021
+# HQ X
+# HQ X
+# HQ X   Edited by Vlad Carare and Lars Schaaf - working as of 07.04.2022
+# HQ X
+# HQ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+# ACRHER2 computer cluster, United Kingdom
+#
+# recommended modules to use with this:
+# - GNU compilers version 9, python and FFTW
+# - obtain these with the following on Archer2
+#
+# module switch PrgEnv-cray PrgEnv-gnu
+# module load PrgEnv-gnu
+# module load gcc/9.3.0
+# module load cray-fftw
+
+
+
+# declarations
+
+include arch/Makefile.linux_x86_64_gfortran
```

Comment:
If you include the Makefile.archer2 here (30) instead you can remove all duplicate declarations and only keep the mpi specific ones.

Reply:
Indeed I agree. I was trying to be consistent with the other makefiles in this repo ([Makefile.linux_x86_64_gfortran_openmpi+openmp](https://github.com/libAtoms/QUIP/blob/public/arch/Makefile.linux_x86_64_gfortran_openmpi%2Bopenmp)).

I'll modify as you said